lib/modules/platform/github/massage-markdown-links.spec.ts

```diff
@@ -87,4 +87,20 @@ describe('modules/platform/github/massage-markdown-links', () => {
       expect(massageMarkdownLinks(input)).toEqual(output);
     }
   );
+
+  it('replace url contains disappear text inline', () => {
+    const input = [
+      'pnpm rebuild accepts --store-dir by @&#8203;user in https://github.com/foo/bar/issues/1\n' +
+        'pnpm rebuild accepts --store-dir by @&#8203;UsEr in https://github.com/foo/bar/issues/2\n' +
+        'pnpm rebuild accepts --store-dir by @&#8203;user-name in https://github.com/foo/bar/issues/3',
+    ].join('\n');
+    const res = massageMarkdownLinks(input);
+    expect(res).toEqual(
+      [
+        'pnpm rebuild accepts --store-dir by `@user` in [foo/bar#1](https://togithub.com/foo/bar/issues/1)\n' +
```

Comment:
The upstream changelog links to the user profiles, so we should try to keep the links in. 😉 

### Link to user profile via `github.com`

Maybe this still causes pings?

```markdown
[`@HonkingGoose`](https://github.com/HonkingGoose)
```

### Link to user profile via `togithub.com` redirect

Or we could replace the `github.com` domain with our `togithub.com` domain to be sure we don't ping back:

```markdown
[`@HonkingGoose`](https://togithub.com/HonkingGoose)
```

Reply:
@HonkingGoose: https://github.com/MaronHatoum/16921/pull/48